src/app/_variables.scss

```diff
@@ -147,3 +141,51 @@ produces
 @function dim-rgb-values-to-rgba($values, $alpha: 1) {
   @return #{'rgba(' + $values + ', ' + $alpha + ')'};
 }
+
+// Theme variables
+:root {
+  // Font
+  --theme-font-primary: 'Open Sans';
+
+  // Base colours
+  --theme-accent-primary: #e8a534;
```

Comment:
Is there any documentation/guidance on branding? 

IMO it's risky to tie style definitions (i.e. naming) to the brand because this can paint yourself into a corner when it comes to ensuring your interaction styles have enough flexibility to work across all view contexts.

I'd recommend something in between:

```scss
--theme-DIM-brand: #e8a534; // this could be defined elsewhere so it's never changed 

--theme-accent-primary: var(--theme-DIM-brand);
--theme-accent-secondary: #68a0b7;
```

This would allow better de-coupling of cosmetic styles (can always use the brand colour) from interaction styles (would use the accents). 

Respectfully, I'd push back on mandating orange as the primary interaction colour for every theme in future. I'd argue that DIM's strongest brand attributes are it's utility and performance — not the orange. You can still easily maintain a permanent visual reference to the orange in the logo + fav icon (which i'd argue is the thing that people see the most when juggling browser tabs) — even if other themes were to use different colour ways.

Reply:
Thanks - I agree with your thinking and I like the idea of having it be a default but not the only option. I think I'm mostly just nervous about themes that make DIM feel boring.